src/PowerShell/Microsoft.WinGet.Client/Common/WinGetIntegrity.cs

```diff
@@ -26,6 +26,18 @@ internal static class WinGetIntegrity
         /// <param name="expectedVersion">Expected version.</param>
         public static void AssertWinGet(PSCmdlet psCmdlet, string expectedVersion)
         {
+            // In-proc shouldn't have other dependencies and thus should be ok.

```

Comment:
>// In-proc shouldn't have other dependencies and thus should be ok. [](http://example.com/codeflow?start=12&length=67)

In-proc requires unpackaged vc redistributable. Do we want to or have a good way to check that?

Reply:
This was caveat 1 from my PR summary.  We will need to statically link the runtime in to be able to release this in a self-contained form.  My hope is that we can do that with some form of property override on the internal release build.  If that doesn't work, then a future PR will need to create a new build flavor that only changes to static linkage.